gammapy/spectrum/spectrum_analysis.py

```diff
@@ -40,6 +46,33 @@ def main(args=None):
     run_spectral_fit_using_config(config)
 
 
+def filter_reflected_regions_analysis(analysis, n_min):
```

Comment:
Make this a method on `gammapy.spectrum.SpectrumAnalysis`?

I know having more modularity and not aggregating all functionality in `*Analysis` is good,
but if the standalone function takes one single `*Analysis` object as input to do something,
there's no advantage over making it a method, no?
(I'd claim the disadvantage is that it's harder to find / understand for users if such things are split into separate functions in the `gammapy.spectrum` namespace.)